sycl/source/kernel_bundle.cpp

```diff
@@ -372,7 +374,9 @@ bool is_source_kernel_bundle_supported(backend BE, source_language Language) {
   // and it's support is limited to the opencl and level_zero backends.
```

Comment:
Comment for "at the moment OpenCL is the only language supported" should be updated.

Right now these two "acceptable" backends is because they support spirv kernel creation and runtime compilation. CUDA and others are AoT only. Maybe we should put that in the comment too?